ShaderTool/Command/Map.cs

```diff
@@ -390,14 +390,13 @@ class Map {
                 ActorData actorData = JsonConvert.DeserializeObject<ActorData>(File.ReadAllText(actorFilePath));
 
                 // Write size of this block in bytes to file
-                // 2 + 2 * (4 + 8) = 26
-                uint actorDataSize = 26 + (actorData.indexCount + actorData.vertexCount) * 4;
+                // 4*16 + 2 + 2*4 = 74
+                uint actorDataSize = 74 + (uint)(actorData.indexCount + actorData.vertices.Length) * 4u;
```

Comment:
honestly, having specific hardcoded numbers is not good programming practice and both of us know it, so i'd say we should make the 74 a static class constant in the actor class

Reply:
(S.O)